Jenkinsfile

```diff
@@ -91,18 +97,22 @@ pipeline {
                 runCompilerPluginTest()
             }
         }
-        stage('Tests Macos') {
+        stage('Tests Macos - Unit Tests') {
             when { expression { runTests } }
             steps {
                 testAndCollect("packages", "macosTest")
-                testAndCollect("test",     "macosTest")
             }
         }
-        stage('Tests Android') {
+        stage('Tests Android - Unit Tests') {
             when { expression { runTests } }
             steps {
                 testAndCollect("packages", "connectedAndroidTest")
-                testAndCollect("test",     "connectedAndroidTest")
+            }
+        }
+        stage('Integration Tests') {
```

Comment:
This grouping goes against what I think would be the better option for parallelising platform specific parts, but I guess I see the motivation for running this together now.

Reply:
Yes, this needs to be refactored once we start to parallelize builds across multiple machines, but for a single machine this setup makes the JenkinsFile less complicated.